### PR TITLE
Open all issues without selecting them

### PR DESCRIPTION
**Pull request by ahmadassaf**
Using extensively the issues and PRs with combinations of filters, I very often come across the need to open all the filtered PRs or issues. This PR adds a new button to open either all issues if you are in the issues page or all the PRs even if you are not an owner.

> *Important*: I have found as part of this PR a bug in the pageDetect logic for issues list and PR list and is fixed now

Related Issues:

 - [Add the add open all issues or PRs button](https://github.com/sindresorhus/refined-github/pull/1401)
 - [Open all selected issues](https://github.com/sindresorhus/refined-github/pull/1229#issuecomment-377228503)

screen shot 2018-07-11 at 4 50 26 pm
screen shot 2018-07-11 at 4 50 39 pm
screen shot 2018-07-11 at 4 56 06 pm

### CODE REVIEW

**Comment by ahmadassaf** [comment]
As @bfred-it suggested this feature is not implemented as part of the already existing `open-all-selected`

File: source/features/open-all-selected.js [hunk 1]
```diff
@@ -1,46 +1,63 @@
...
 		action: 'openAllInTabs'
 	});
 }
 
 export default function () {
-	if (!isIssueList()) {
+	if (!pageDetect.isIssueList() || select.all('.js-issue-row').length < 2) {
 		return;
 	}
+	const openAllButtonText = location.pathname.endsWith('/issues') ? `Open all issues` : `Open all PRs`;
```
Comment (bfred-it) [inline source/features/open-all-selected.js]: This was removed here https://github.com/sindresorhus/refined-github/pull/1402/commits/692b01b040d30102067f0a908d3ae33bf9f42c29

**Comment by bfred-it** [comment]
Great! Now it only needs a readme mention and screenshot

**Comment by ahmadassaf** [comment]
on it .. shall i rename the file btw instead of `open-all-selected` to something more generic to describe the new functionality `open-issues-and-PRs` ?

**Comment by bfred-it** [comment]
Perhaps `batch-open-issues` works. No need to specify PRs most of the time since they're the same. This would also need a migration in `background.js`

**Comment by bfred-it** [comment]
This is so close to being merged! :)

**Comment by bfred-it** [comment]
I needed this just now 😅

**Comment by bfred-it** [comment]
Thanks for working on this @ahmadassaf! I found it instantly useful

**Pull request merged**